modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,186 @@
-= Section 2
+= Elyra Pipelines
 
-This is _Section 2_ of _Chapter 1_ in the *hello* quick course....
\ No newline at end of file
+Elyra provides a visual pipeline editor for building pipelines from Python and R scripts as well as Jupyter notebooks, simplifying the conversion of multiple files into batch jobs or workflows. A pipeline in Elyra consists of nodes that are connected with each other to define execution dependencies.
+
+image::elyra_jupyter.png[Jupyter with elyra]
```

Comment:
Image not found 
![image](https://github.com/RedHatQuickCourses/rhods-pipelines/assets/675470/674cc154-2071-4399-8eb9-14f879b177be)